src/euphorie/client/browser/templates/login.pt

```diff
@@ -149,6 +149,34 @@
 
           <fieldset class="vertical">
 
+            <label class="${python:'error' if 'first_name' in errors else None}">
+              <i18n:msg translate="label_first_name_optional">First name (optional)</i18n:msg>
+              <input class="pat-autofocus"
+                     name="first_name"
+                     required
```

Comment:
The first and last name fields are proposed as optional but they are actually required
![image](https://user-images.githubusercontent.com/1300763/174775096-be121371-fbe0-442b-8097-41cad7d5c378.png)